### PR TITLE
added hypothesis test case

### PR DESCRIPTION
**Pull request by mmarconi**
# Description of Pull Request

Added in a hypothesis test case.
The test case I wrote uses hypothesis to test the second function in test_group_method.py.

Works #112 

## Added in a test case

Please describe the pull request as one of the following: 

- [ ] Bug fix
- [ ] Breaking change
- [ ] New feature
- [ ] Documentation update
- [x] Other Code Coverage

## Tags

@huangs1 @Michionlion @Lancasterwu 

@libailong

### CODE REVIEW

File: gatorgrouper/tests/test_group_method.py [hunk 1]
```diff
@@ -1,7 +1,18 @@
...
+from hypothesis import Verbosity
+from hypothesis.strategies import integers
+
+# from hypothesis.strategies import text
+import pytest
 from utils import group_random
 from utils import group_rrobin
 
+# import parse_arguments
+# import gatorgrouper
```
Comment (Michionlion) [inline gatorgrouper/tests/test_group_method.py]: There are a number of unneeded comments in this file; they should be removed if they are not needed.

File: gatorgrouper/tests/test_group_method.py [hunk 2]
```diff
@@ -60,6 +71,25 @@ def test_group_random_extra():
     assert num_group == 2
 
 
+@given(grpsize=integers(min_value=1, max_value=3))
+@settings(verbosity=Verbosity.verbose)
+@pytest.mark.hypothesisworks
+def test_group_random2(grpsize):
+    """This hypothesis test will test the test_group_random_extra"""
```
Comment (Michionlion) [inline gatorgrouper/tests/test_group_method.py]: This docstring is incorrect -- this test only tests the `group_random_group_size` method. It is, however, a good model for a hypothesis test.

File: gatorgrouper/tests/test_group_method.py [hunk 3]
```diff
@@ -60,6 +71,25 @@ def test_group_random_extra():
...
+        ["Marvin", False, False, True, True],
+        ["Evin", True, True, True, False],
+        ["Nikki", True, True, False, False],
+        ["Nick", True, False, True, False],
+        ["Dan", False, True, False, True],
+    ]
+    returned_groups = group_random.group_random_group_size(responses, grpsize)
+    size_count = grpsize
+    assert len(returned_groups[0]) == size_count
+    assert grpsize == size_count
```
Comment (Michionlion) [inline gatorgrouper/tests/test_group_method.py]: This second assertion means nothing -- `size_count` was just set to be `grpsize`. You can completely remove the mention of `size_count`. You should also decide if you need to check for more than just the size of the first group (`returned_groups[0]`). Do all groups have to have the same size to pass this test? What is the intended behavior for `group_random_group_size`?

File: gatorgrouper/tests/test_group_method.py (hunk 2 above)
Comment (mmarconi) [reply gatorgrouper/tests/test_group_method.py]: I changed the docstring.

File: gatorgrouper/tests/test_group_method.py (hunk 3 above)
Comment (mmarconi) [reply gatorgrouper/tests/test_group_method.py]: I deleted the second assertion. I will consult with my team about about checking the size of more than just the first group.

**Pull request merged**